Emby.Server.Implementations/Localization/LocalizationManager.cs

```diff
@@ -159,29 +150,20 @@ private void LoadRatings(string country, ParentalRating[] ratings)
             _allParentalRatings[country] = ratings.ToDictionary(i => i.Name);
         }
 
-        private List<string> GetRatingsFiles(string directory)
-        {
-            return _fileSystem.GetFilePaths(directory, false)
-                .Where(i => string.Equals(Path.GetExtension(i), ".txt", StringComparison.OrdinalIgnoreCase))
-                .Where(i => Path.GetFileName(i).StartsWith("ratings-", StringComparison.OrdinalIgnoreCase))
-                .ToList();
-        }
+        private IEnumerable<string> GetRatingsFiles(string directory)
+            => _fileSystem.GetFilePaths(directory, false)
+                .Where(i => string.Equals(Path.GetExtension(i), ".csv", StringComparison.OrdinalIgnoreCase))
```

Comment:
Why rename `.txt` to `.csv`? Just curious.

Reply:
They are csv this and way my editor shows them properly :p